src/components/Navigation.css

```diff
@@ -0,0 +1,98 @@
+.nav-container {
+  width: 100%;
+  height: 92px;
+  background-color: rgb(236, 236, 245);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  box-shadow: 2px 2px 12px rgb(109, 107, 107);
```

Comment:
- [ ] wonderful job so far. please like the previous reviewer mentioned, we are to follow the zeplin design as much a possible. kindly use the original color `#0090ff` for the update buttons and also the add book button has a thesame color as well. nice work. 
yours
![Screenshot_39](https://github.com/richie1988/My_Bookstore_App/assets/65298048/d057ea02-f99a-40d7-8d9e-3989702cee19)
original
![Screenshot_37](https://github.com/richie1988/My_Bookstore_App/assets/65298048/9babb592-257d-4870-a0e4-537bb5decc63)